src/services/tourettes.js

```diff
@@ -62,12 +62,24 @@ function getProfanities(amount) {
   return profanityReply.join(' ');
 }
 
+function isInt(data) {
+  if (typeof data === 'number' && (data % 1) === 0) {
+    return true;
+  }
+  return false;
+}
+
 export function sayTourettes(inputText = '') {
+  const reply = spewTourettes(inputText.split(' '));
+
   if (inputText === '') {
     return getProfanities(getRandomNumber(3, 10));
   }
 
-  const reply = spewTourettes(inputText.split(' '));
+  if (isInt(+inputText)) {
```

Comment:
Absolutely, passing constants and variables as arguments is better in every way, just trying to get it through my skull.